cnm/plugin/main.go

```diff
@@ -158,9 +159,15 @@ func main() {
 		return
 	}
 
+	processLockCli, err := processlock.NewFileLock(platform.CNILockPath + name + store.LockExtension)
```

Comment:
pls no globals, these should be config that's injected (after reasonable defaults are set if necessary)

Reply:
this value is different from previous and we cannot have one common global var.

Reply:
Not sure I understand. I see something similar to (platform.CNILockPath + pluginName + store.LockExtension) in 5 places in this code and this path will never change (right?) because lockPath, pluginName and extension are 'static'.

Reply:
pluginName will change based on service